draft-ietf-tls-tls13.md

```diff
@@ -2612,6 +2625,11 @@ facilitates implementation as described below). Servers MUST check
 that it is the last extension and otherwise fail the handshake with an
 "illegal_parameter" alert.
 
+When the syntax of this extension is correct but the server cannot
+verify at least one binder because of some configuration or state
+changes, the server SHOULD fall back to a full 1-RTT handshake if a
+"key_share" extension is supplied.
```

Comment:
Ah, I wrote this PR with confusion. The binder MUST be verified anyway.
What I should have talked about the encrypted PSK value itself.
I will push a fix.
Thank you for pointing out.